File: dayOne/java/levelOne/PalindromeArray.java
public class PalindromeArray {

    public static boolean checkIfPalindrome(int[] array) {

        int left = 0;
        int right = array.length - 1;

        while(left < right) {
            if (array[left] != array[right])
                return false;

            left++;
            right--;
        }

        return true;
    }
}

File: dayOne/javaScript/levelOne/palindromeArray.js

const isPalindrome = (numbers) => {
    let left = 0;
    let right = numbers.length - 1;

    while (left < right) {
        if (numbers[left] != numbers[right])
            return false;

        left++;
        right--;
    }

    return true;
};


module.exports = { isPalindrome };